### PR TITLE
docs(landingpage): add lottie animation

### PR DESCRIPTION
**Pull request by SoonIter**
## Summary

maybe I need to extract the lottie json files to https://github.com/rspack-contrib/rstack-design-resources?

#### Before

image

#### After

image

### CODE REVIEW

File: website/theme/components/Landingpage/WhyRspack/useLottieAnimation.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    console.log(animationRef.current);
+  }, [lottieJsonUrl]);
+  useEffect(() => {
+    if (!animationRef.current || !ref.current) {
+      return;
+    }
+
+    if (isHovering) {
+      animationRef.current.goToAndPlay(0, true);
+      console.log(111111);
```
Comment (chenjiahan) [inline website/theme/components/Landingpage/WhyRspack/useLottieAnimation.tsx:37]: remove log

File: website/theme/components/Landingpage/WhyRspack/useLottieAnimation.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+    const animation = lottie.loadAnimation({
+      container: ref.current,
+      animationData: lottieJsonUrl,
+      renderer: 'svg',
+      loop: false,
+      autoplay: false,
+    });
+
+    animation.setSpeed(1.4);
```
Comment (chenjiahan) [inline website/theme/components/Landingpage/WhyRspack/useLottieAnimation.tsx:25]: the animation can be faster, especially the “Fast Startup" one

**Review by chenjiahan — COMMENTED** [review]
Why is the bundle size getting smaller..

image

**Comment by chenjiahan** [comment]
> maybe I need to extract the lottie json files to https://github.com/rspack-contrib/rstack-design-resources

These JSONs can be placed with the component code, as we need to import them.

**Comment by SoonIter** [comment]
> Why is the bundle size getting smaller..

sorry 😸, invert

File: website/theme/components/Landingpage/WhyRspack/useLottieAnimation.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      animationData: lottieJsonUrl,
+      renderer: 'svg',
+      loop: false,
+      autoplay: false,
+    });
+
+    animation.setSpeed(3);
+
+    animationRef.current = animation;
+    console.log(animationRef.current);
```
Comment (chenjiahan) [inline website/theme/components/Landingpage/WhyRspack/useLottieAnimation.tsx:28]: remove log

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**